proc_Hahn_echoph.py

```diff
@@ -1,67 +1,45 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping
-fl = figlist_var()
+from proc_scripts import *
+from proc_scripts.load_data import postproc_lookup
+from sympy import symbols
+fl = fl_mod()
 t2 = symbols('t2')
 filter_bandwidth = 5e3
-for date,id_string,label_str in [
-        ('191007','echo_2','n'),
+color_choice = True
+for searchstr,exp_type,nodename,postproc,label_str,color_str in [
+        ('200302_alex_probe_water','test_equip','signal','Hahn_echoph','microwaves off','blue'),
         ]:
-    filename = date+'_'+id_string+'.h5'
-    nodename = 'signal'
-    s = nddata_hdf5(filename+'/'+nodename,
-            directory = getDATADIR(
-                exp_type = 'test_equip'))
-    nPoints = s.get_prop('acq_params')['nPoints']
-    nEchoes = s.get_prop('acq_params')['nEchoes']
-    nPhaseSteps = 8 
-    SW_kHz = s.get_prop('acq_params')['SW_kHz']
-    nScans = s.get_prop('acq_params')['nScans']
-    print(ndshape(s))
-    s.reorder('t',first=True)
-    t2_axis = s.getaxis('t')[0:2048]
-    s.setaxis('t',None)
-    s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
-    s.setaxis('ph2',r_[0.,2.]/4)
-    s.setaxis('ph1',r_[0.,1.,2.,3.]/4)
-    s.set_units('t2','s')
-    s.reorder('t2',first=False)
-    s.ft('t2',shift=True)
-    fl.next('ft')
-    fl.plot(s)
-    s = s['t2':(-2e3,2e3)]
-    s.ift('t2')
-    fl.next('raw data')
-    fl.image(s)
-    fl.next('raw data -- coherence pathways')
-    #s = s['power',0].C
-    s.ft(['ph2','ph1'])
-    #s *= exp(-s.getaxis('t2')/10e-3)
-    fl.image(s)
-    fl.show();quit()
-    fl.next('filtered + rough centered data')
-    s.ft('t2', shift=True)
-    s *= exp(-1j*2*pi*10.3)
-    s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
+    s = find_file(searchstr, exp_type=exp_type, expno=nodename,
+            postproc=postproc, lookup=postproc_lookup)
+    s.mean('nScans')    
+    fl.next('signal')
+    slice_f = (-5e3,5e3)
+    s = s['t2':slice_f]
     s.ift('t2')
     rough_center = abs(s).convolve('t2',0.01).mean_all_but('t2').argmax('t2').item()
     s.setaxis(t2-rough_center)
-    fl.image(s)
     s.ft('t2')
-    k = s.C
+    s_apodized = s.C*exp(-1j*s.fromaxis('t2')*0.9*2*pi)
+    s_apodized = s.C*exp(-1j*0.9*2*pi)
+    s_apodized *= exp(-1j*s_apodized.fromaxis('t2')*2*pi*0.005)
+    s = s_apodized.C
```

Comment:
hard-coded phase adjustments.  also, it's not clear what s_apodized is doing

Reply:
see above